lerna.json

```diff
@@ -1,4 +1,7 @@
 {
-  "lerna": "2.0.0-beta.32",
-  "version": "independent"
+  "lerna": "2.0.0-beta.37",
+  "version": "independent",
+  "packages": [
```

Comment:
Because lerna changed its settings during version bump, it's good practice to run lerna init. That's what I did and that's why there is packages property